programs/rodeos/rocksdb_plugin.cpp

```diff
@@ -59,7 +55,8 @@ std::shared_ptr<chain_kv::database> rocksdb_plugin::get_db() {
       ilog("rodeos database is ${d}", ("d", my->db_path.string()));
       if (!bfs::exists(my->db_path.parent_path()))
          bfs::create_directories(my->db_path.parent_path());
-      my->database = std::make_shared<chain_kv::database>(my->db_path.c_str(), true, my->threads, my->max_open_files);
+
+      my->database = std::make_shared<chain_kv::database>(my->db_path.c_str(), true, std::nullopt, std::nullopt, my->options_file_name);
```

Comment:
`my->threads` & `my->max_open_files` and args to `chain_kv::database` should all be removed.

Reply:
chain_kv::database is instantiated in embedded_rodeos.cpp too. I was worried removing the arguments might break existing functionalities. But a further look shows the default values are 0s. It should be be OK to just use system defined defaults. Will change.